scripts/windows/deploy.one.cmd

```diff
@@ -34,8 +34,6 @@ REM
     set rdst=\\%machine%\%rdst_dir%
     @mkdir %rdst%    
     xcopy /F /Y /S %src_dir% %rdst%
-    COPY /Y %bin_dir%\7z.exe %rdst%
```

Comment:
it is in the publish script - but you are right, should 7z be copied in publish script instead.